spec/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
- [ ] **Required:** Just as the previous reviewers have stated, kindly ensure that all files in the spec folder, i.e test files end with spec. For example `comment_spec.rb`

Reply:
I have fix it